### ISSUE TITLE
Package request: pycharm-professional

### ISSUE THREAD

**Issue by jdobonb** [void-linux/void-packages#48245]
### Package name

pycharm-professional

### Package homepage

https://www.jetbrains.com/pycharm/

### Description

The closed sourced version of PyCharm, currently the only way it can be used is with Flatpak and a more native aproach will be benecifial. I suppose it will need to be restricted in the same way the `intellij-idea-ultimate-edition` package is.

### Does the requested package meet the package requirements?

System

### Is the requested package released?

Yes

**Comment by chrysos349** [comment]
> the only way it can be used is with Flatpak

 not really. you can install any jetbrains products via their toolbox app ( https://www.jetbrains.com/toolbox-app/ ).

**Comment by kutu-dev** [comment]
@chrysos349 that is still far from ideal, as far as I can tell it doesn't make system-wise installs and puts all the files in a subdirectory in `~/.local`. Maybe adding the toolbox itself as a pacakage would be a good middleground for making easier to install all the JetBrains products, specially the closed-sourced ones.
